### PR TITLE
Update the Quickstart

### PR DESCRIPTION
**Pull request by jkatz**
Now includes a "very, VERY quickstart" but drives people to use the
container-based installer.

### CODE REVIEW

File: docs/content/quickstart/_index.md [hunk 1]
```diff
@@ -11,200 +11,120 @@ Can't wait to try out the PostgreSQL Operator? Let us show you the quickest poss
...
-
-For an OpenShfit deployment, you will at a minimum have to to uncomment and set the `openshift_host` variable. This is the location of where your OpenShift environment is, and can be obtained from your administrator. For example:
-
-```python
-openshift_host="https://openshift.example.com:6443"
-```
+### Configure the PostgreSQL Operator Installer
 
-Based on how your OpenShift environment is configured, you may need to set the following variables:
+There are many [configuration parameters](<{{< relref "/installation/configuration.md">}}>) to help you fine tune your installation, but there are a few that you may want to change to get the PostgreSQL Operator to run in your environment. Open up the `postgres-operator.yml` file and edit a few variables.
```
Comment (jmckulk) [inline docs/content/quickstart/_index.md:52]: I think this link should be `({{< relref "/installation/configuration.md">}})`

File: docs/content/quickstart/_index.md [hunk 2]
```diff
@@ -11,200 +11,120 @@ Can't wait to try out the PostgreSQL Operator? Let us show you the quickest poss
...
+  value: "nfsstorage"
+- name: BACKUP_STORAGE
+  value: "nfsstorage"
+- name: PRIMARY_STORAGE
+  value: "nfsstorage"
+- name: REPLICA_STORAGE
+  value: "nfsstorage"
 ```
 
+For a full list of available storage types that can be used with this installation method, please review the [configuration parameters](<{{< relref "/installation/configuration.md">}}>).
```
Comment (jmckulk) [inline docs/content/quickstart/_index.md:71]: same here

File: docs/content/quickstart/_index.md [hunk 3]
```diff
@@ -257,10 +177,10 @@ pgo-apiserver version 4.3.0
 
 ## Step 5: Have Some Fun - Create a PostgreSQL Cluster
```
Comment (jmckulk) [inline docs/content/quickstart/_index.md:178]: step 4?

File: docs/content/quickstart/_index.md (hunk 1 above)
Comment (jkatz) [reply docs/content/quickstart/_index.md:52]: Yup, thanks!

**Comment by jmckulk** [comment]
The namespace will need to exist for the install to work for both quick starts. Maybe we should add a step saying to create it, either that or add the assumption that it exists. 

Those two links still look broken if you render the file through github

**Comment by jmckulk** [comment]
I pulled down the branch and used hugo to render the docs and those links do work 👍

File: docs/content/quickstart/_index.md [hunk 4]
```diff
@@ -11,200 +11,120 @@ Can't wait to try out the PostgreSQL Operator? Let us show you the quickest poss
...
-replica_storage='rook'
+- name: BACKREST_STORAGE
+  value: "nfsstorage"
+- name: BACKUP_STORAGE
+  value: "nfsstorage"
+- name: PRIMARY_STORAGE
+  value: "nfsstorage"
+- name: REPLICA_STORAGE
+  value: "nfsstorage"
 ```
```
Comment (jmckulk) [inline docs/content/quickstart/_index.md:70]: Looks like this got added twice

Comment (jkatz) [reply docs/content/quickstart/_index.md:70]: How so? This seems OK to me.

File: docs/content/quickstart/_index.md [hunk 5]
```diff
@@ -11,200 +11,120 @@ Can't wait to try out the PostgreSQL Operator? Let us show you the quickest poss
...
 
+For a full list of available storage types that can be used with this installation method, please review the [configuration parameters]({{< relref "/installation/configuration.md">}}).
```
Comment (jmckulk) [inline docs/content/quickstart/_index.md:71]: Sorry i'm bad at git hub. Should be line 71 and 73
```For a full list of available storage types that can be used with this installation method, please review the configuration parameters.

For a full list of available storage types that can be used with this installation method, see: $URL
```

**Comment by jkatz** [comment]
Merged in 70e9e5639cce2f62b27f0de1b02ff0d2aebf959a and 4dc75820e1a9d7488dc48dddcf5538a67a1cd23b

**Pull request closed without merging**